frontend/src/services/applicationService.js

```diff
@@ -24,6 +24,17 @@ export default {
     }
   },
 
+  async getRedirectedApplicationByFacilityId(facilityId) {
```

Comment:
Could we save some redundant code in the calling components by passing in a list of facility ids?

Reply:
yup - done!